{{cookiecutter.project_repo_name}}/Makefile

```diff
@@ -2,8 +2,8 @@
 APP_ROOT := $(abspath $(lastword $(MAKEFILE_LIST))/..)
 APP_NAME := {{ cookiecutter.project_slug }}
 
-WPS_URL = http://localhost:{{ cookiecutter.http_port }}/
-OUTPUT_URL = $(WPS_URL)/wpsoutputs
+WPS_URL = http://localhost:{{ cookiecutter.http_port }}
+OUTPUT_URL = https://pavics.ouranos.ca/wpsoutputs
```

Comment:
I would not use a hard coded output-url. We can add this parameter to the cookiecutter variables with a default.

Reply:
`OUTPUT_URL` currently is only used in `refresh-notebooks` target (https://github.com/bird-house/cookiecutter-birdhouse/blob/7c1e712b5ed609802e37a35878d5cd6d5f133c65/%7B%7Bcookiecutter.project_repo_name%7D%7D/Makefile#L132-L135) for the notebooks to not fail Jenkins when testing against the production deployment and for the notebooks to serve as tutorial notebooks against production deployment as well.

It is a make variable so it can be overridden when invoking make when needed.  Essentially, it is already a default value that can be overridden.

Would adding a comment clarifying the intention help?

Reply:
A comment would be helpful ... if it is only an optional make task.

But are these parameters not related to the values in the pywps configuration?
https://github.com/bird-house/cookiecutter-birdhouse/blob/90aa958fbcd10f1e77d0636a05fe29d744831d27/%7B%7Bcookiecutter.project_repo_name%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/default.cfg#L11

Reply:
What do you mean an optional make task?  Optional in the sense Travis-CI do not run it?  Then yes.

It indeed is related to the that pywps config you linked.

When running locally for testing, the default output_url is `http://localhost:port/outputs` as you linked.  But in production it is `https://pavics.ouranos.ca/wpsoutputs`.  So after batch refreshing all the notebooks, we simply make it look like it has been refreshed from the production instance but in fact it has been refreshed from the most up-to-date local devel instance.